.github/workflows/build-images-action.yml

```diff
@@ -8,49 +8,24 @@ on:
     tags:
     - 'v*'
 
-permissions: {}
```

Comment:
Put this back.

Reply:
Unfortunately that wouldn't work. The wf would fail with:

```
The workflow is not valid. .github/workflows/build-images-action.yml (Line: 15, Col: 3): Error calling workflow 'Nordix/metal3-project-infra/.github/workflows/container-image-build.yml@mquhuy/change-container-image-build-wf-permissions'. The nested job 'job' is requesting 'contents: read', but is only allowed 'contents: none'.
```

Reply:
https://github.com/metal3-io/baremetal-operator/actions/runs/10300338972

Reply:
Or maybe we could put `permissions: contents: read` as the default instead of `permissions: {}`?

Reply:
OK, it seems that is the working option. Let's do that